NethermindNode.Tests/Helpers/FuzzerHelper.cs

```diff
@@ -13,7 +13,7 @@ public static void Fuzz(FuzzerCommandOptions fuzzerCommandOptions, NLog.Logger l
 
 public class FuzzerCommandOptions : IFuzzerCommand
 {
-    public string DockerContainerName { get; set; } = "sedge-execution-client";
+    public string DockerContainerName { get; set; } = ConfigurationHelper.Instance["execution-container-name"];
```

Comment:
Do we need a setter here?

Reply:
Valid point - default is set on Fuzzer level and here default should not be applied.